_includes/releases/v22.1/v22.1.1.md

```diff
@@ -20,7 +16,6 @@ Release Date: June 6, 2022
 <h3 id="v22-1-1-sql-language-changes">SQL language changes</h3>
 
 - `ttl_job_cron` is now displayed on [`SHOW CREATE TABLE`](../v22.1/show-create.html) and the specified `reloptions` by default. [#80292][#80292]
-- Added the `crdb_internal.cluster_locks` virtual table, which exposes the current state of locks on keys tracked by concurrency control. The virtual table displays metadata on locks currently held by transactions, as well as operations waiting to obtain the locks, and as such can be used to visualize active contention. The `VIEWACTIVITY` or `VIEWACTIVITYREDACTED` [role option](../v22.1/security-reference/authorization.html#role-options), or the [`admin` role](../v22.1/security-reference/authorization.html#admin-role), is required to access the virtual table; however, if the user only has the `VIEWACTIVITYREDACTED` role option, the key on which a lock is held will be redacted. [#80517][#80517]
```

Comment:
This should not be removed. It's being documented in this PR: https://github.com/cockroachdb/docs/pull/14598